src/js/tabs/led_strip.js

```diff
@@ -870,25 +859,24 @@ TABS.led_strip.initialize = function (callback, scrollPosition) {
 
         // set directions visibility
         if (semver.lt(FC.CONFIG.apiVersion, "1.20.0")) {
-            switch (activeFunction) {
-                case "function-r":
-                    $('.indicatorOverlay').hide();
-                    $('.directions').hide();
-                break;
-                default:
-                    $('.indicatorOverlay').show();
-                    $('.directions').show();
-                break;
+            if (activeFunction) {
```

Comment:
From looking through the code, it seems to be possible for `activeFunction` to take other values than `function-r`, so these are not equivalent.